test/e2e/node/security_context.go

```diff
@@ -146,33 +146,29 @@ var _ = SIGDescribe("Security Context", func() {
 		testPodSELinuxLabeling(f, false, true)
 	})
 
-	ginkgo.It("should support seccomp alpha unconfined annotation on the container [Feature:Seccomp] [LinuxOnly]", func() {
-		// TODO: port to SecurityContext as soon as seccomp is out of alpha
+	ginkgo.It("should support seccomp alpha unconfined annotation on the container [LinuxOnly]", func() {
```

Comment:
change the descriptions to drop "annotation"

Reply:
Yep fxed, dropped the `alpha` as well. :+1: